src/pmdas/openvswitch/pmdaopenvswitch.py

```diff
@@ -148,7 +148,7 @@ def __init__(self, name, domain):
 
     def fetch_switch_info(self):
         """ fetches result from command line """
-        out = subprocess.Popen([self.pmda_dir+'/switch.sh'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+        out = subprocess.Popen([self.pmda_dir+'/openvswitch/switch.sh'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
One more thing I noticed - I am nitpicking here - it might be worthwhile to use OS specific separator in file paths where it's relatively less time intensive to implement like here. I am not sure about state of PCP on Windows, but making the agent little bit future proof shouldn't do any harm. Opinion @natoscott ?

Reply:
Ah thanks again. I didn't think of this. Will resolve ASAP.